micro_sam/util.py

```diff
@@ -109,35 +113,9 @@ def models():
             "vit_b_em": "https://zenodo.org/record/8250260/files/vit_b_em.pth?download=1",
         },
     )
-    # This extra dictionary is needed for _get_checkpoint to download weights from a specific checkpoint
-    models.download_names={
-            "vit_h": "vit_h.pth",
-            "vit_l": "vit_l.pth",
-            "vit_b": "vit_b.pth",
-            "vit_t": "vit_t_mobile_sam.pth",
-            "vit_h_lm": "vit_h_lm.pth",
-            "vit_b_lm": "vit_b_lm.pth",
-            "vit_h_em": "vit_h_em.pth",
-            "vit_b_em": "vit_b_em.pth",
-        }
     return models
 
 
-def _get_checkpoint(model_name, checkpoint_path):
```

Comment:
This is not needed anymore. We easier use the model downloaded by pooch and stored in the cache directory, or we use a local weight file.

In the second case we do not copy that weight file or compare its hash.

Reply:
👍